fsck/fsck.c

```diff
@@ -383,7 +383,7 @@ static int boot_region_checksum(struct exfat *exfat)
 	__le32 checksum;
 	unsigned short size;
 	void *sect;
-	int i;
+	size_t i;
```

Comment:
if int is replaced with unsigned int, warnings are generated? size_t is also unsigned long in some system.